integration_tests/models/source/snowflake/snowplow_ecommerce_events_stg.sql

```diff
@@ -0,0 +1,22 @@
+-- page view context is given as json string in csv. Parse json
+with prep as (
+select
+  *,
+  parse_json(contexts_com_snowplowanalytics_snowplow_web_page_1_0_0) as contexts_com_snowplowanalytics_snowplow_web_page_1,
+  parse_json(contexts_com_snowplowanalytics_snowplow_ecommerce_user_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_user_1,
+  parse_json(contexts_com_snowplowanalytics_snowplow_ecommerce_page_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_page_1,
+  parse_json(unstruct_event_com_snowplowanalytics_snowplow_ecommerce_snowplow_ecommerce_action_1_0_0) as unstruct_event_com_snowplowanalytics_snowplow_ecommerce_snowplow_ecommerce_action_1,
+  parse_json(contexts_com_snowplowanalytics_snowplow_ecommerce_cart_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_cart_1,
+  parse_json(contexts_com_snowplowanalytics_snowplow_ecommerce_product_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_product_1,
+  parse_json(contexts_com_snowplowanalytics_snowplow_ecommerce_checkout_step_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_checkout_step_1,
+  parse_json(contexts_com_snowplowanalytics_snowplow_ecommerce_transaction_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_transaction_1
+
+
+from {{ ref('snowplow_ecommerce_events') }}
+)
+
+
+select
+  *
+
+from prep
```

Comment:
This doesn't seem to match what we do with other snowflake integration tests with contexts? Maybe we over complicated it before and no longer need it. If you do need it, then take a look [here](https://github.com/snowplow/dbt-snowplow-mobile/blob/1ebd8f76884e83acd09e3e3433283e9606a0be37/integration_tests/models/source/snowflake/snowplow_mobile_events_stg.sql#L430) for how I built a context with potentially null values for some of the fields.